apps/andi/assets/css/app.scss

```diff
@@ -23,4 +24,65 @@ body {
   margin: 1em;
 }
 
+@mixin area($block, $element) {
+  .#{$block}__#{$element} {
+    grid-area: $element
```

Comment:
What does this do? I'm not familiar with this scss syntax.

Reply:
`#{$var}` = string interpolation (I assume that's what's confusing about it)
It will generate this:
```
.my-block__my-element {
  grid-area: my-element;
}
```
If you look in edit.scss you can see where I called it a ton and make that value judgement about whether the complexity is worth the space saved (I'm expecting us to use css grid a lot in this app so it should have utility outside of this set of changes).